extensions/amp-auto-lightbox/0.1/amp-auto-lightbox.js

```diff
@@ -61,16 +61,28 @@ export const RENDER_AREA_RATIO = 1.2;
 export const VIEWPORT_AREA_RATIO = 0.25;
 
 /**
- * Ancestors considered "actionable", i.e. that are bound to a default onclick
- * action (e.g. `button`) or where it cannot be determined whether they're
- * actionable or not (e.g. `amp-script`).
+ * Selector for subnodes for which the auto-lightbox treatment does not apply.
  */
-const ACTIONABLE_ANCESTORS =
+const DISABLED_ANCESTORS =
+    // Runtime-specific.
+    '[placeholder],' +
+
+    // Explicitly opted out.
+    '[data-amp-auto-lightbox-disable],' +
+
+    // Semantically annotated.
+    '[role=presentation],' +
```

Comment:
This one is debatable. Where did you notice this being an issue?

Most cases this is used for icons, but certain cases like images that show data/graphs, this is used to hide it from AT and provide an alternative off-screen `table` or `text` to convey the same data as the image.

Reply:
SGTM, I don't feel strongly about it, so removed.